src/core/router/info.h

```diff
@@ -241,6 +242,9 @@ struct RouterInfoTraits
         case Trait::IntroKey:
           return "ikey";
 
+        case Trait::IntroExp:
+          return "iexp";
```

Comment:
Thought about implementing as well, but in the spec for proposal 133, the key is `iexp{0,1,2}`. This won't match, will it?

The only other way I saw with our current implementation was a somewhat nasty `Trait::IntroExp0: return "iexp0";` and so on... It seemed better to ignore.

Reply:
> This won't match, will it?

It will. Look at lines 340-347 in router/info.cc

Reply:
FYI, it's not literally `iexp{0,1,2}`.

Reply:
I know, was just using that for shorthand. I'll check the lines in router/info.cc